### DISCUSSION TITLE
Chat channel, or set up system chat room

### DISCUSSION THREAD

**Discussion by SheerYin** [Zrips/CMI#6237] [Feature Requests]
The bungee server needs to distinguish between player chats
Such as skyblock/survival, so that the chat becomes clean 

By creating a chat channel or system chat room (will not disappear)

**Comment by mrfloris** [comment]
Just to share information, there's `/cmi chat` which can be used to create temporary and persistant chat rooms. 
They are not bound to worlds (would be nice)

**Reply by SheerYin** [reply]
cmi chat cannot support bungee, and it cannot remain permanent
Restart the server and it will be gone

**Reply by mrfloris** [reply]
Nonsense.

**Reply by SheerYin** [reply]
Because I have multiple survivals (survival1, survival2) they need to share chat
There are or multiple skyblocks, they also need to share chat
However, only similar channels can achieve this effect

We used cmi to format the chat for a long time, relatively satisfied, and don’t want to switch to other chat plug-ins
